sdk/formrecognizer/ai-form-recognizer/src/options/FormRecognizerClientOptions.ts

```diff
@@ -68,6 +68,12 @@ export interface FormRecognizerCommonClientOptions extends CommonClientOptions {
    * Default: FormRecognizerApiVersion.Latest ("2021-09-30-preview")
    */
   apiVersion?: FormRecognizerApiVersion;
+  /**
+   * Gets or sets the audience to use for authentication with Azure Active Directory.
```

Comment:
The doc string would be clearer if it communicated that it is only necessary to override this setting if you're 1. using AAD/TokenCredential and 2. are using a cloud other than Azure Public.